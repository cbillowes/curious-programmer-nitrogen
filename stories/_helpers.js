import React from 'react'
import Layout from '../src/components/layout'
import { SmallSpacer } from '../src/components/spacer'

const Page = ({ className, children }) => {
  return (
    <Layout
      className={className}
    >
      <SmallSpacer />
      {children}
    </Layout>
  )
}

export default Page