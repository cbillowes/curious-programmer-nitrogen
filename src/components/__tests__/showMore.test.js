import React from 'react'
import ShowMore from '../showMore'
import { getSnapshot } from './_helpers'

describe(`Show more items`, () => {
  it(`should render`, () => {
    const showMore = (
      <ShowMore
        to="/archives"
      />
    )
    const tree = getSnapshot(showMore)
    expect(tree).toMatchSnapshot()
  })
})
