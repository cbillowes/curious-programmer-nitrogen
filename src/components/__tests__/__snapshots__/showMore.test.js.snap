// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show more items should render 1`] = `
<div
  className="show-more"
>
   
  <a
    href="/archives"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-ellipsis-h fa-w-16 "
      data-icon="ellipsis-h"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M328 256c0 39.8-32.2 72-72 72s-72-32.2-72-72 32.2-72 72-72 72 32.2 72 72zm104-72c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72zm-352 0c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </a>
   
</div>
`;
